gateway/dataimpl/server_v1/collectionadminserver.go

```diff
@@ -158,6 +160,20 @@ func (s *CollectionAdminServer) CreateCollection(
 		return nil, s.errorHandler.NewGenericStatus(err).Err()
 	}
 
+	manifestUid, err := strconv.ParseUint(resp.ManifestUid, 16, 64)
+	if err != nil {
+		return nil, s.errorHandler.NewGenericStatus(err).Err()
+	}
+
+	opts := gocbcorex.EnsureManifestOptions{
+		BucketName:    in.BucketName,
+		CollectionUid: manifestUid,
```

Comment:
I don't think this is necessarily needed for this review but this naming is massively confusing. Does `EnsureManifestOptions` in gocbcorex need updating to have ManifestUid not CollectionUid?

Reply:
Oops.  That was definitely my error.  Sorry @Westwooo

Reply:
Fixed in a pending PR for gocbcorex: https://github.com/couchbase/gocbcorex/pull/207